2/greeter.rb

```diff
@@ -3,5 +3,5 @@
 
 name=gets.chomp
 
-puts 'Hello, ' + name + "!" do  
+puts 'Hello, ' + name.upcase + "!" do  
```

Comment:
Nice! How did you find this?

Reply:
I remembered .upcase from the Makers' 'learn Ruby in 20 minutes' introduction